src/test/resources/application-test.yaml

```
@@ -33,4 +33,10 @@ naver:
     client-id: ${NAVER_CLIENT_ID:dummy-id}
     client-secret: ${NAVER_CLIENT_SECRET:dummy-secret}
     ocr-secret: ${NAVER_OCR_SECRET:dummy-ocr-secret}
-    ocr-url: ${NAVER_OCR_URL:https://dummy-ocr-url.com}
\ No newline at end of file
+    ocr-url: ${NAVER_OCR_URL:https://dummy-ocr-url.com}
+
+logging:
+  level:
+    root: INFO
+    com.twogether.deokhugam: INFO
+    org.springframework.test: INFO
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)